### ISSUE TITLE
Dies when can't parse? "object reference not set to an instance of an object"

### ISSUE THREAD

**Issue by ChubbBear** [adultmm/AdultMediaManager#81]
i.e.
![image](https://user-images.githubusercontent.com/42285946/220490193-dd24c5f5-68d4-4645-8684-3bd9364640dd.png)

![image](https://user-images.githubusercontent.com/42285946/220489126-356ab31b-6bb1-4145-b5e5-c31aebff6d42.png)

![image](https://user-images.githubusercontent.com/42285946/220489186-bc53ed7a-9fc4-4b25-9ae2-e00acefa99a9.png)

In scraping a title, while AMM can parse some sites, **it can entirely die** from just one bad one... sans an option to select a different title from that site or cancel (skip) the site.

Perhaps ref?: https://stackoverflow.com/questions/779091/what-does-object-reference-not-set-to-an-instance-of-an-object-mean

**Comment by adultmm** [comment]
Could you send me the log file, please?

**Comment by ChubbBear** [comment]
Yup, please see attached.

I think you too may notice that it dies at AdultScraper_Image (and just with Jeedoo only, that I've noticed).

Notably, just out of curiosity at a latter point, I disabled the dll (scraper.Image.AdultScraper.dll) and ran the scrape (sans scraping the image) and the program did not crash. Plus I was able to select another option in the Jeedoo results. No images scraped, but at least the data pulled through.

Cheers!

[2023-02-21.csv](https://github.com/adultmm/AdultMediaManager/files/10854161/2023-02-21.csv)

**Comment by chronotank79** [comment]
I also have this exact issue. I can also confirm, adding".bak" to the end of scraper.Image.AdultScraper.dll prevents the crash, but also reduces the images the scraper is able to find. I also checked Jeedoo for the title in question, and I noticed that they have 2 entries in their database for the same movie - one of them is a dead link that says something like "this title no longer available", while the other one has all the data as normal.  My theory is that maybe the scraper is checking the bad page, which makes it choke.

It would be good if there was a way to just disable scraping from Jeedoo. I found an option in the settings to choose which sites get scraped, but Jeedoo was already disabled, and the scraper seems to always try to scrape Jeedoo regardless of which setting you pick. It also wouldn't be an issue if the Jeedoo scraper failing didn't make the whole program crash.

I don't run into this error for 99% of movies I try to scrape, but when I do I can't find any workaround so far. Very frustrating.
